Refactored Functions

ShawnToubeau opened the pull request:
Refactored createLiveLinkTable and createUserFavoriteTable into one function createTable

Commit:
Refactored createLiveLinkTable and createUserFavoriteTable into one function createTable

Commit:
Git saved my butt once again

Ethan-Arrowood commented:
This looks great Shawn! Some feedback:
- package-lock.json should't be changed here. I recommend deleting it and the node_modules folder again and reinstalling the packages through `npm install`. You'll need to commit the newly generated file after you do this. (And if it is still different from what I have that is okay and I'll figure it out 👍 )
- keep your commit messages shorter. I think the (unenforced) limit is 50 characters. This is purely so that when I see your commit message in browser I can read the whole thing. If you want to add more information I believe there is a way to have a 'new line' and then you can write however much you want... but the first line should be 50 char or less. (and also this is just a recommendation not a hard rule)

Keep up the awesome work 😄

Commit:
Fresh npm install

ShawnToubeau commented:
Tried your suggestions, package-lock.json is still modified 🤔

Ethan-Arrowood commented:
Lol somehow it added more changes to the package-lock

Ethan-Arrowood committed:
Removing package-lock

I'm removing this file manually. There is a fair amount of dep inconsistencies in my branch so I'll figure it out next.
